tests/DCPS/FooTest3_0/PubDriver.cpp

```diff
@@ -173,6 +174,7 @@ PubDriver::initialize(int& argc, ACE_TCHAR *argv[])
                                    ::OpenDDS::DCPS::DEFAULT_STATUS_MASK);
   TEST_CHECK (! CORBA::is_nil (publisher_.in ()));
 
+  OpenDDS::DCPS::RestoreOutputStreamState ross(std::cout);
```

Comment:
Shouldn't there a scope so that the options are directly set back after the write?